packages/core/integration-tests/test/transpilation.js

```diff
@@ -183,6 +183,18 @@ describe('transpilation', function () {
     assert(file.includes('_jsxRuntime.jsx("div"'));
   });
 
+  it('should support the automatic JSX runtime with preact with alias', async function () {
+    let b = await bundle(
+      path.join(
+        __dirname,
+        '/integration/jsx-automatic-preact-with-alias/index.js',
+      ),
+    );
+
+    let file = await outputFS.readFile(b.getBundles()[0].filePath, 'utf8');
+    assert(file.includes('_jsxRuntime.jsx("div"'));
```

Comment:
I think we should also add this to make sure it's actually importing from React (using a Regex because `react/jsx-runtime` is a substring of `preact/jsx-runtime`).

```suggestion
    let file = await outputFS.readFile(b.getBundles()[0].filePath, 'utf8');
    assert(/\Wreact\/jsx-runtime\W/.test(file));
    assert(file.includes('_jsxRuntime.jsx("div"'));
```

Reply:
Sure. I fixed in 4efc78 :+1: Thanks!